### PR TITLE
Use i18n extractor

### PR DESCRIPTION
**Pull request by mbarbet**

### CODE REVIEW

**Review by sebbousquet — APPROVED** [review]

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -1,39 +1,67 @@
...
+	"Apply geo-queries": "Apply",
+	"Set geo-queries": "Set",
+	"Manage layers": "Manage layers",
+	"show_layers": "Show/hide these layers",
+	"Attachments": "Attachments",
+	"ASC": "Ascendant",
+	"DESC": "Descendant",
+	"Sort on": "Sort on",
+	"Layers and style": "Layers and style",
+	"Informations": "Informations",
```
Comment (MohamedHamouGisaia) [inline src/assets/i18n/en.json:26]: Informations has no 's'

File: src/assets/i18n/fr.json [hunk 2]
```diff
@@ -1,40 +1,67 @@
...
+	"GeoJson": "GeoJson",
+	"WKT": "WKT",
+	"Paste your WKT": "Coller ici votre WKT",
+	"Zoom to result": "Zoomer sur le résultat",
+	"Cancel": "Fermer",
+	"Import": "Importer",
+	"Geo-queries & basemaps": "Requêtes géographiques et fonds de carte",
+	"Geo-queries": "Requêtes géographiques",
+	"operation": "opération",
+	"point or geometry": "",
```
Comment (MohamedHamouGisaia) [inline src/assets/i18n/fr.json:14]: I would translate it : "géometrie"

File: src/assets/i18n/fr.json [hunk 3]
```diff
@@ -1,40 +1,67 @@
...
+	"Basemaps": "Fonds de carte",
+	"Choose one of the following basemaps": "",
```
Comment (MohamedHamouGisaia) [inline src/assets/i18n/fr.json:16]: "Choisir un fond de carte

File: src/assets/i18n/fr.json [hunk 4]
```diff
@@ -1,40 +1,67 @@
...
+	"ASC": "Ascendent",
+	"DESC": "Descendent",
+	"Sort on": "Tri par",
+	"Layers and style": "Couche et style",
+	"Informations": "Informations",
+	"Choose layer": "Choisir une couche",
+	"Choose style": "Choisir un style",
+	"Choose date": "Choisir une date",
+	"Choose": "Choisir",
+	"Unable to preview this style for this layer": "Impossible de prévisualiser cette style pour cette couche.",
```
Comment (MohamedHamouGisaia) [inline src/assets/i18n/fr.json:31]: ce style

File: src/assets/i18n/fr.json [hunk 5]
```diff
@@ -1,40 +1,67 @@
...
+	"Click to show details": "Cliquer pour afficher le détail",
+	"Geo-sort": "Geo-tri",
+	"Geo sort action": "Tri par rapport à la distance au centre de la carte",
+	"Timeout": "Temps d'attente dépassé",
+	"Problem parsing input file": "Erreur dans le parcours du fichier d'entrée",
+	"Only `zip` file is allowed": "Seulement les fichiers `zip` sont acceptés",
+	"Zip file must contain at least a `*.shp`, `*.shx` and `*.dbf` or a `*.json`": "Le fichier zip doit contenir au minimum un fichier `*.shp`, `*.shx` et `*.dbf` OU un fichier `*.json`",
+	"Too many vertices in a polygon": "Un polygone contient trop de sommets",
+	"No polygon to display in this file": "Aucun polygone à afficher",
+	"Too much features": "Le fichier contient trop d'elements",
```
Comment (MohamedHamouGisaia) [inline src/assets/i18n/fr.json:62]: éléments

**Review by MohamedHamouGisaia — APPROVED** [review]

**Pull request merged**